src/poliastro/tests/test_czml.py

```diff
@@ -1,21 +1,32 @@
-import pytest  # noqa: E402 isort:skip
+"""
```

Comment:
Sorry, I told you to comment the tests but I don't think it's necessary to add module and function docstrings. Tests should be simple and self-explanatory as much as possible, except for comments for weird cases or important decisions

Reply:
pylint was complaining about missing docstrings. Do you want them to be removed?

Reply:
yes please! if anything, for consistency with the rest (I prefer removing this than having docstrings everywhere in the tests)

Reply:
Removed.